bool isDisGlobalMap
float32 distance
float resolution
---
bool success
